commands/generate-key.js

```diff
@@ -0,0 +1,17 @@
+const KeyPair = require('nearlib').KeyPair;
+const exitOnError = require('../utils/exit-on-error');
+
+module.exports = {
+    command: 'generate-key <account-id>',
+    desc: 'generate key ',
+    builder: (yargs) => yargs,
+    handler: exitOnError(async (argv) => {
+        let near = await require('../utils/connect')(argv);
+        if (argv.accountId) {
+            const { deps: { keyStore }} = near.config;
+            const keyPair = KeyPair.fromRandom('ed25519');
+            await keyStore.setKey(argv.networkId, argv.accountId, keyPair);
+            console.log(`Generated key pair with ${keyPair.publicKey} public key`);
+        }
+    })
+};
```

Comment:
why do we have this function individually not with other functions? and this makes command and function together? why not follow what we already have?

Reply:
@icerove didn't want to do too many chnages at once to pull every function, so for now only pulled new functions like this

The reason to do this is that it's more convenient to work with such structure if there are concurrent modifications that needs to be merged (e.g. we both add some commands).
Also current structure avoids extra step (you just define a command at once, without having to define a function first and then wrap it into command).